pkg/actors/actor.go

```diff
@@ -56,15 +57,18 @@ type actor struct {
 	once sync.Once
 }
 
-func newActor(actorType, actorID string, maxReentrancyDepth *int) *actor {
+func newActor(actorType, actorID string, maxReentrancyDepth *int, now *time.Time) *actor {
```

Comment:
Why not pass a `clk clock.Clock` and use `clk.Now().UTC()` instead of passing a pointer with the current time? This can be used to update `lastUsedTime` in `lock` too.

Reply:
Good call, fixed